### PR TITLE
UITest and screenshot for Accompanying Certificates

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Add UITest and screenshot for Accompanying Certificates

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13094

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/AccompanyingCertificates/AccompanyingCertificatesViewController.swift [hunk 1]
```diff
@@ -33,7 +33,8 @@ class AccompanyingCertificatesViewController: DynamicTableViewController, Dismis
 		super.viewDidLoad()
 
 		navigationItem.rightBarButtonItem = dismissHandlingCloseBarButton
-		
+		navigationItem.rightBarButtonItem?.accessibilityIdentifier = AccessibilityIdentifiers.HealthCertificate.Reissuance.accompanyingCertificatesCloseButton
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/AccompanyingCertificates/AccompanyingCertificatesViewController.swift:36]: It looks to me like the `dismissHandlingCloseBarButton ` already has the accessibility identifier `AccessibilityIdentifiers.AccessibilityLabel.close` by default, I don't think you need to introduce a new one here

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService+UITestingMethods.swift [hunk 2]
```diff
@@ -258,14 +282,51 @@ extension HealthCertificateService {
...
+						action: "renew"
+					)
+				]
 			),
 			certificatesRevokedByInvalidationRules: dccWalletInfo.certificatesRevokedByInvalidationRules
 		)
 	}
+	private var firstReissuanceBase45: String {
+		"HC1:6BFOXN*TS0BI$ZDFRH5TSWK3*93X+01V80RBZEJ8USXG43NI 7EFXRV7P4XUS4FDMP7JM:UC*GP-S4FT5D75W9AV88E34L/5R3F9JAY-BI9EW+9K.CMIAO/B.OC4-9D.B6LFDC9 AE 2AB7LOKEH-BXPLV6I-TL8KES/F-1JZ.KBIH:HG43MWY8%FAGUU0QIRR97I2HOAXL92L0: KQMK8J4RK46YBBOAXM42%K:XFEK0OH6PO9:TUQJAJG9-*NIRICVELZUZM9EN9-O97LAQCK /P/T14SIWG9$R7064B69X5QNEQT56QHE+.6L80TM8$M8SNCNAK09TU*5KGIDPS7A8Q/DA-7TLOJ3KV:BW.7-$4UGGBEG*Z1W1O4WDQETN:523EDZT/P7+7S/ZEJZ9Q:NIBV84W1-HGN33SNVMAS6C4XL QQU6S9QLJ KX$FTDAM2H3TS3KNY%AI:OF/9NHB%3KKS06GBF E"
+	}
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService+UITestingMethods.swift:317]: Couldn't these just be `let`s?

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService+UITestingMethods.swift (hunk 2 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService+UITestingMethods.swift:317]: @nickguendling  'let' declarations cannot be computed properties

**Review by marcussc — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/AccompanyingCertificates/AccompanyingCertificatesViewController.swift (hunk 1 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/AccompanyingCertificates/AccompanyingCertificatesViewController.swift:36]: Just to document our discussion: There are two close buttons, so we need to introduce another identifier here to ensure the top-most one is tapped

**Pull request merged**